release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -208,6 +208,14 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 * An IPv6 network for API and Ingress services on the second `machineNetwork` configuration.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+[id="ocp-4-16-networking-metallb-frr-k8s_{context}"]
+==== Integration of MetalLB and FRR-K8s (Technology Preview)
+
+This release introduces `FRR-K8s`, a Kubernetes based DaemonSet that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
```

Comment:
See the repo guidelines for guidance on [general](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) vs [object](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) formatting for APIs. _DaemonSet_ should be formatted as either "daemon set" or `DaemonSet`.

Reply:
Thanks for the review @michaelryanpeter